src/libraries/System.Private.Xml/src/System/Xml/Core/XmlParserContext.cs

```diff
@@ -11,16 +12,16 @@ namespace System.Xml
     // Specifies the context that the XmLReader will use for xml fragment
     public class XmlParserContext
     {
-        private XmlNameTable _nt = null;
-        private XmlNamespaceManager _nsMgr = null;
+        private XmlNameTable? _nt = null;
+        private XmlNamespaceManager? _nsMgr = null;
         private string _docTypeName = string.Empty;
         private string _pubId = string.Empty;
         private string _sysId = string.Empty;
         private string _internalSubset = string.Empty;
         private string _xmlLang = string.Empty;
         private XmlSpace _xmlSpace;
         private string _baseURI = string.Empty;
-        private Encoding _encoding = null;
+        private Encoding? _encoding = null;
 
         public XmlParserContext(XmlNameTable nt, XmlNamespaceManager nsMgr, string xmlLang, XmlSpace xmlSpace)
```

Comment:
From doc for all overloads `XmlNameTable nt, XmlNamespaceManager nsMgr, string xmlLang` are all nullable

Reply:
the constructor has nullable args but I think fields are annotated correctly since constructor `?? string.Empty` for almost all args. The nt in nsMgr I believe have relationship that one of them must not be null although that makes them both nullable and I don't think we have any attribute or way to express that